docs/world maintainer.md

```diff
@@ -0,0 +1,54 @@
+# World Maintainer
+
+A world maintainer is a person responsible for a world or part of a world in Archipelago.
+
+If a world author does not want to take on the responsibilities of a world maintainer, they can release their world as
+a "private" [APWorld](/docs/apworld%20specification.md) or maintain their own fork instead.
```

Comment:
```suggestion
a unofficial [APWorld](/docs/apworld%20specification.md) or maintain their own fork instead.
```

Reply:
should change `a unofficial` to `an unofficial` if you go with this